cpp/include/DataStorm/InternalI.h

```diff
@@ -142,7 +142,6 @@ namespace DataStormI
         virtual ~FilterFactory() = default;
 
         virtual std::shared_ptr<Filter> get(std::int64_t) const = 0;
-        virtual std::shared_ptr<Filter> decode(const Ice::CommunicatorPtr&, const Ice::ByteSeq&) = 0;
```

Comment:
This was never called for a FilterFactoryT. But the implementation was preventing us for passing the name and lambda in the FilterT constructor.